hack/verify-release.sh

```diff
@@ -630,7 +635,7 @@ verify_vulnerabilities()
     go_version="$(make go-version)"
     echo "GoVersionOverride = \"${go_version}\"" > "${config_file}"
     "${OSVSCANNER_CMD[@]}" scan \
```

Comment:
Fails for me with

```
Failed to read config file: open .osv-scanner.toml: permission denied
open .osv-scanner.toml: permission denied
```

Reply:
Huh, this has not changed? Whats your environment/setup?

Reply:
It's Fedora 41 with podman. Where does this file come from?

Reply:
It is written on L636, ie. one line above. It is really weird if it can write it but cannot read it back. 

That means when mounted to container (I assume you're now using podman), there is some security mechanism that isn't present in docker.  Does something force a uid change?

Like said, this isn't changed, so it must be something changed in your environment since you last tried the script (and backported to IRSO) few months back?

Reply:
> That means when mounted to container (I assume you're now using podman), there is some security mechanism that isn't present in docker

SELinux would do it potentially..

Reply:
Yeah, it was SELinux. You can apply a diff like this to make it work:

```
diff --git a/hack/verify-release.sh b/hack/verify-release.sh
index 49309354..084ed7d7 100755
--- a/hack/verify-release.sh
+++ b/hack/verify-release.sh
@@ -151,6 +151,13 @@ if [[ -n "${CONTAINER_RUNTIME}" ]]; then
     declare -a OSVSCANNER_CMD=(
         "${CONTAINER_RUNTIME}" run --rm
         -v "${PWD}":/src -w /src
+    )
+    if [[ "${CONTAINER_RUNTIME}" == podman ]]; then
+        OSVSCANNER_CMD+=(
+            --security-opt=label=disable
+        )
+    fi
+    OSVSCANNER_CMD+=(
         ghcr.io/google/osv-scanner:v2.0.0@sha256:ceea4d7c57dcb4ab65453445f7a3155d0cc9ccef66a098a516ac264677d4b61f
     )
 else
```

Reply:
Not necessarily. With :z you're going to relabel local files, which then is going to cause problems for local users.